reInvent18_Developer_Workshop/FILTER2D_Lab.md

```diff
@@ -21,7 +21,7 @@ The kernel used in this tutorial is a 2D video filter, a function widely used in
 
 To simplify this tutorial, a workspace preloaded with the initial project configuration is provided in the lab repository.
 
-Please also note that although the entire tutorial is performed on an F1 instance, only the final step of this tutorial really needs to be run on F1. All the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as C4. However, to avoid switching from C4 to F1 instances during this tutorial, all the steps are performed on the F1 instance.
+Please also note that although the entire tutorial is performed on an F1 instance, only the final step of this tutorial really needs to be run on F1. All the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as T3. However, to avoid switching instances during this tutorial, all the steps are performed on the F1 instance.
```

Comment:
Hi Gadi,

"ll the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as T3."

can we change above to: "all the interactive development, profiling and optimization steps  can be performed  on a cost-effective AWS EC2 instance such as T3 or any other instance that is supported by AWS FPGA Developer AMI. Alternately, Development can also be done using on-premise systems that has Xilinx VIVADO  & SDx Tools installed."

Idea is to present AWS as first choice for development and highlights the availability of FPGA Developer AMI that comes with free Vivado toolset.